service/history/workflow/mutable_state.go

```diff
@@ -360,9 +360,8 @@ type (
 			baseRunLowestCommonAncestorEventID int64,
 			baseRunLowestCommonAncestorEventVersion int64,
 		)
-		UpdateActivity(*persistencespb.ActivityInfo) error
-		UpdateActivityWithTimerHeartbeat(*persistencespb.ActivityInfo, time.Time) error
-		UpdateActivityWithCallback(string, UpdateActivityCallback) error
+		UpdateActivity(string, ActivityUpdater) error
```

Comment:
I think all other activity related update methods takes in scheduledEventID as input, ideally we should keep them consistent.

Reply:
input parameter changed to scheduledEventId